Gem/CMakeLists.txt

```diff
@@ -6,6 +6,4 @@
 #
 #
 
-set(ASV_OPENXR_SAMPLE_ENABLED FALSE CACHE BOOL "Enable the OpenXR Gems and samples in Atom Sample Viewer")
-
```

Comment:
Ignore this change.  I accidentally pushed this cache variable to the `openxr` branch while I was updating the branch with latest `development`.